public/assets/css/actuality.css

```diff
@@ -0,0 +1,139 @@
+main {
+    background-image: url("../images/empreintechienblanc.png");
+    background-color: var(--light);
+    background-repeat: repeat-y;
+    background-size: 35%;
+    background-position: 80% 30%;
+    padding-top: calc(var(--spacing) * 2);
+    --text-shadow: 0px 3px 5px rgb(68, 66, 66, 0.5);
+}
+
+section > div {
+    display: grid;
+    gap: var(--spacing);
+    grid-template-columns: 1fr;
+}
+
+section > header > h1 {
+    font-size: clamp(1rem, 4vw, calc(var(--font-size) * 1.5));
+    color: var(--primary);
+    font-weight: 600;
+    font-family: var(--font-title);
+    text-align: center;
+}
+
+article {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    width: clamp(360px, 70%, 95vw);
+    margin: var(--spacing);
+    --stagger-delay: 150ms;
+    animation: 400ms FadeInAnimation var(--stagger-delay) ease-out;
+    animation-fill-mode: backwards;
+    animation-iteration-count: 1;
+}
+
+article > header {
+    margin-bottom: calc(var(--block-spacing-vertical) * 1 / 3);
+    padding: var(--spacing);
+    background-image: url("../images/dog-information.webp");
+    background-repeat: no-repeat;
+    background-size: 10%;
+    background-position: 97% 30%;
+    background-color: var(--light);
+}
+
+article > header > h2 {
+    margin-bottom: 0;
+    font-size: clamp(0.9rem, 1.9vw, var(--font-size));
+    color: var(--secondary);
+    font-weight: 600;
+    text-shadow: var(--text-shadow);
+}
+
+article > footer {
+    margin-top: calc(var(--block-spacing-vertical) * 2 / 3);
+    padding: 0 0 0 calc(var(--spacing) * 2 / 3);
+    color: var(--light);
+    background-color: var(--primary);
+}
+
+footer a[role="button"] {
+    width: clamp(6rem, 10vw, 8rem);
+    padding: calc(var(--spacing) / 3) calc(var(--spacing) / 3);
+    border-radius: 0;
+    text-shadow: var(--text-shadow);
+    font-size: calc(var(--font-size) * 3 / 4);
+}
+
+article > footer {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0;
+}
+
+article:nth-child(even) {
+    justify-self: end;
+}
+
+article:nth-child(odd) {
+    justify-self: start;
+}
+
+article:nth-child(even) > header {
+    text-align: right;
+    padding-right: 15%;
+}
+
+article:nth-child(odd) > header {
+    background-position: 3% 30%;
+    text-align: left;
+    padding-left: 15%;
+}
+
+article:nth-child(even) > footer {
+    flex-direction: row-reverse;
+    padding-inline-end: 2vw;
+    background-image: linear-gradient(
+        to right,
+        var(--intermadiate) 30%,
+        var(--primary) 70%
+    );
+}
+
+article:nth-child(odd) > footer {
+    padding-inline-start: 2vw;
+    background-image: linear-gradient(
+        to left,
+        var(--intermadiate) 30%,
+        var(--primary) 70%
+    );
+}
```

Comment:
des répétitions, une idée de refacto: 

article > footer {
    flex-direction: var(--flex-direction, row-reverse);
    padding-inline: var(--padding-start, var(--spacing)) var(--padding-end, 0);

    background-image: linear-gradient(
        to var(--direction, right),
        var(--intermadiate) 30%,
        var(--primary) 70%
    );
}

article:nth-child(odd) > footer {
    --direction: right;
    --padding-start: var(--spacing);
    --padding-end: 0;
    --flex-direction: row;
)